packages/xforms-engine/src/lib/reactivity/createSelectItems.ts

```diff
@@ -73,25 +85,20 @@ const createSelectItemsetItemLabel = (
 	context: EvaluationContext,
 	definition: ItemsetDefinition,
 	itemValue: Accessor<string>
-): Accessor<TextRange<'label'>> => {
+): Accessor<ClientTextRange<'item-label'>> => {
 	const { label } = definition;
 
 	if (label == null) {
 		return createMemo(() => {
```

Comment:
If you're asking whether it should just reference the value rather than producing a reactive function: it has to be computed, because itemset item values are themselves computed expressions.

If you're asking whether it should be a bare thunk/`Accessor` versus `createMemo`, I don't know whether it would matter (probably not in this case). I don't expect there's very much overhead associated with Solid's `createMemo` (unlike similar APIs in other frameworks, where there might be diffing involved), so if it's unnecessary it's probably harmless. But I'm happy to look deeper to say with more certainty whether it's strictly necessary, if you'd prefer.

Reply:
yes I was asking about `createMemo`. If it is harmless then let's keep it.